src/Renderer/Camera.js

```diff
@@ -37,148 +48,226 @@ function updatePreSse(camera, height, fov) {
     // => replacing horizontalFOV in Math.tan(horizontalFOV * 0.5)
-    // Math.tan(horizontalFOV * 0.5) = Math.tan(2 * Math.atan(this.width * 0.5 / focale) * 0.5)
-    //                               = Math.tan(Math.atan(this.width * 0.5 / focale))
-    //                               = this.width * 0.5 / focale
-    // => now replacing focale
+    // Math.tan(horizontalFOV * 0.5) = Math.tan(2 * Math.atan(this.width * 0.5 / focal) * 0.5)
+    //                               = Math.tan(Math.atan(this.width * 0.5 / focal))
+    //                               = this.width * 0.5 / focal
+    // => now replacing focal
     //                               = this.width * 0.5 / (this.height * 0.5) / Math.tan(verticalFOV * 0.5)
     //                               = Math.tan(verticalFOV * 0.5) * this.width / this.height
     // => back to (1)
     // horizontalPreSSE = this.width / (2.0 * Math.tan(verticalFOV * 0.5) * this.width / this.height)
     //                  = this.height / 2.0 * Math.tan(verticalFOV * 0.5)
     //                  = verticalPreSSE
+
     if (camera.camera3D.isOrthographicCamera) {
         camera._preSSE = height;
     } else {
-        camera._preSSE = verticalPreSSE;
+        const verticalFOV = THREE.MathUtils.degToRad(fov);
+        camera._preSSE = height / (2.0 * Math.tan(verticalFOV * 0.5));
     }
 }
 
 /**
  * Wrapper around Three.js camera to expose some geographic helpers.
  *
- * @param   {string}                crs                                     The camera's coordinate projection system
- * @param   {number}                width                                   The width (in pixels) of the view the camera is
- * associated to.
- * @param   {number}                height                                  The height (in pixels) of the view the camera is
- * associated to.
- * @param   {object}                [options]                               Options for the camera.
- * @param   {THREE.Camera}          [options.cameraThree]                   A custom Three.js camera object to wrap around.
- * @param   {Camera~CAMERA_TYPE}    [options.type=CAMERA_TYPE.PERSPECTIVE]  The type of the camera. See {@link CAMERA_TYPE}.
- * @param   {number}                [options.orthoExtent=50]                The height of the extent that a camera from type
- * `CAMERA_TYPE.ORTHOGRAPHIC` must cover initially.
- * @constructor
+ * @property    {string}    crs                             The camera's coordinate projection system.
+ * @property    {number}    aspect                          The ratio `width/height` of the camera.
+ * @property    {object}    camera3D                        The Three.js camera that is wrapped around.
+ * @property    {number}    width                           The width of the camera.
+ * @property    {number}    height                          The height of the camera.
+ * @property    {boolean}   matrixProjectionNeedsUpdate     True if Three.js camera needs an update of its projection
```

Comment:
`matrixProjectionNeedsUpdate`
could you remove it, because it's used in itowns core only.